test/functional/Java8andUp/src_80/org/openj9/test/java/lang/Test_Class.java

```diff
@@ -377,6 +377,70 @@ public Object run() {
 		java.security.AccessController.doPrivileged(action);
 	}
 
+	static class ParentClass {
+		public static void methodPublicStatic() {}
+		public void methodPublicInstance() {}
+		static void methodPkgPrivateStatic() {}
+		void methodPkgPrivateInstance() {}
+		private static void methodPrivateStatic() {}
+		private void methodPrivateInstance() {}
+	}
+
+	static class ChildClass extends ParentClass {
+		public static void methodPublicStatic() {}
+		public void methodPublicInstance() {}
+		static void methodPkgPrivateStatic() {}
+		void methodPkgPrivateInstance() {}
+		private static void methodPrivateStatic() {}
+		private void methodPrivateInstance() {}
+	}
+
+	static interface SuperInterface {
+		void methodPublicInterface(); 
+		public static void methodPublicStatic() {};
+	}
+
+	static interface SubInterface extends SuperInterface {
+		void methodPublicInterface(); 
```

Comment:
Can you add a default method to the interface?  Maybe two - one overridden by the class and one not.

Reply:
Added a `default method` overridden by the `class`, this test intends to verify that more specific methods (i.e., child class methods overriding parent methods) are returned by `j.l.Class.getMethods()`. 
Current tests treat any method declared by parent class/interface as a failure since all parent methods are overridden. Adding a parent method not overridden by the child class makes the verification less straightforward, and doesn't add more value to this PR either.
`Test_Class` already has `test_getMethods_subtest1()` and `test_getMethods_subtest2()` to verify the methods returned from `java.lang.Class#getMethods()`.